web/themes/custom/fifth_sub_bootstrap/src/scss/custom/general/_views_view_fields.scss

```
@@ -0,0 +1,51 @@
+.views-field-name {
+  @include rfs(-1.75rem, margin-top);
+  line-height: 2;
+  span {
+    background-color: $main-purple;
+    padding: 0.125rem 0.6rem;
+    @include rfs(0.625rem);
+    letter-spacing: 1px;
+    a {
+      color: $main-white;
+    }
+  }
+}
+.views-field-title {
+  line-height: 1.1;
+  @include rfs(1.0625rem);
+  font-weight: 700;
+  margin: 0.5rem 0 0 0;
+  a {
+    color: $title-default;
+  }
+}
+.views-field-uid, .views-field-changed, .views-field-created, .views-field-revision-timestamp, .views-field-changed-1 {
```

This is not a critical issue, but it's better to split up the different cases (classes)
```suggestion
.views-field-uid,
.views-field-changed,
.views-field-created,
.views-field-revision-timestamp,
.views-field-changed-1 {
```